smart-contracts/test/RelayPool/fees.hardhat.ts

```diff
@@ -71,7 +68,7 @@ describe('Fees', () => {
       10,
       relayBridgeOptimism
     )
-    const fees = (amount * bridgeFee) / 10000n
+    const fees = (amount * bridgeFee) / 100000000n // Using fractional basis points
```

Comment:
Let's use the constant here so that test works even if we use a different denomination!